frontend/appflowy_flutter/lib/mobile/presentation/base/mobile_view_page.dart

```diff
@@ -114,33 +114,39 @@ class _MobileViewPageState extends State<MobileViewPage> {
 
   Widget _buildApp(ViewPB? view, List<Widget> actions, Widget child) {
     final icon = view?.icon.value;
-    return Scaffold(
-      appBar: AppBar(
-        titleSpacing: 0,
-        title: Row(
-          mainAxisSize: MainAxisSize.min,
-          children: [
-            if (icon != null)
-              EmojiText(
-                emoji: '$icon ',
-                fontSize: 22.0,
-              ),
-            Expanded(
-              child: FlowyText.regular(
-                view?.name ?? widget.title ?? '',
-                fontSize: 14.0,
-                overflow: TextOverflow.ellipsis,
+    return WillPopScope(
+      // prevent navigate to previous screen by scrolling left on the screen
+      onWillPop: () async {
```

Comment:
It's definitely not a good solution to disable the swipe-left gesture.

You should add padding between the swipe-left gesture hit test area and the scrollable widget area instead.